internal/usage/usage.go

```diff
@@ -117,7 +117,8 @@ dbName and collection are only required for built-in roles.`
 	StartDate                                 = "Timestamp in ISO 8601 date and time format in UTC when the maintenance window starts."
 	EndDate                                   = "Timestamp in ISO 8601 date and time format in UTC when the maintenance window ends."
 	AuthResult                                = "Flag that indicates whether to return either successful or failed authentication attempts. When set to success, Atlas filters the log to return only successful authentication attempts. When set to fail, Atlas filters the log to return only failed authentication attempts."
-	AccessLogDate                             = "Timestamp in the number of milliseconds that have elapsed, since the UNIX Epoch, for the first entry that Atlas returns from the database access logs."
+	AccessLogStartDate                        = "Timestamp in the number of milliseconds that have elapsed, since the UNIX Epoch, for the first entry that Atlas returns from the database access logs."
```

Comment:
need to `make gen-docs`

Reply:
@blva, thanks! Can you reapprove. When I run `golangci-lint`, I get the following warning:
`WARN [linters_context] rowserrcheck is disabled because of generics. You can track the evolution of the generics support by following the https://github.com/golangci/golangci-lint/issues/2649.` Is that an issue?

Reply:
nope since our PR lint check is working, thanks!